src/App.tsx

```diff
@@ -23,19 +24,21 @@ function App () {
     {
       path: "/",
       element: <DashboardScreen auth={auth} />,
-      errorElement: <ErrorPage />
+      errorElement: <ErrorScreen />
     }
   ]);
 
   return (
     <div className="App">
-      <AuthenticationProvider
-        auth={auth}
-        setAuth={setAuth}
-        userProfile={userProfile}
-        setUserProfile={setUserProfile}>
-        <RouterProvider router={router} />
-      </AuthenticationProvider>
+      <ErrorHandler>
+        <AuthenticationProvider
+          auth={auth}
+          setAuth={setAuth}
+          userProfile={userProfile}
+          setUserProfile={setUserProfile}>
```

Comment:
If jotai is installed then shouldn't these props be accessed as atoms within the AuthenticationProvider

Reply:
Made the props accessible within the AuthenticationProvider using Jotai.